buildres/linux/jabrefHost.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python -u
```

Comment:
I'd set this to
`#!/usr/bin/env python3 -u`
(do we need the -u? it sets the output to be unbuffered I think)
So it's properly set to python3
and not hardcoded to a path (https://docs.python.org/3/using/unix.html)

Reply:
I'd still change this line, to be sure that we use python3